redirects.csv

```diff
@@ -1,7 +1,11 @@
 http://${env:CRDS_APP_DOMAIN}/*,https://${env:CRDS_APP_DOMAIN}/:splat,301!
+/fun/,/not-fun/,301
+/cool/,/coolest/,302
+/about,/about-us,302
+/contact,/contact-us,301
```

Comment:
Unless you want these to go to production, you should consider pulling them out.